gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=io.ballerina.scan
-version=0.9.1-SNAPSHOT
+version=0.10.0
```

Comment:
Should be the SNAPSHOT version?

Reply:
Changed with 47e3da03cb290c7bfd182cd1fb7e93c09e94964e